index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title> Rock , Paper, Scisscors </title>
+</head>
+
+<body>
+    <h1> Rock , Paper, Scisscors </h1>
+    <p> Welcome to the game of Rock, Paper, Scisscors </p>
+    <p> Please choose one of the following: </p>
+    <button id="rock"> Rock </button>
+    <button id="paper"> Paper </button>
+    <button id="scissors"> Scissors </button>
+ 
+    <<script src="script.js" defer></script>
```

Comment:
Double "<" at the beginning, a all the script has to be moved in the head, just below the <title> tag - I've sent you a screenshot from the last lecture :)